docs/onebusaway-gtfs-merge-cli.md

```
@@ -2,111 +2,129 @@
-https://repo1.maven.org/maven2/org/onebusaway/onebusaway-gtfs-merge-cli/3.2.2/onebusaway-gtfs-merge-cli-3.2.2.jar
+https://repo1.maven.org/maven2/org/onebusaway/onebusaway-gtfs-merge-cli/14.0.0/onebusaway-gtfs-merge-cli-14.0.0.jar
 
 ## Using the Application
 
-You'll need a Java 21 runtime installed to run the cli. 
+The current build targets **Java 25**, so you need a Java 25 (or newer) runtime to run a jar built
+from this source. (Older released jars were built against older Java versions.)
 
 To run the application:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)